src/App.js

```diff
@@ -1,5 +1,70 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+import {
+  inputCarsException,
+  inputCountException,
+} from "./exceptionHandling.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const cars = await inputCars();
+      const count = await inputCount();
+      const result = await racingCount(count, cars);
+
+      Console.print("최종 우승자 : " + getWinners(cars, result));
+    } catch (error) {
+      throw new Error(`[ERROR] ${error.message}`);
+    }
+  }
 }
 
+const inputCars = async () => {
+  const inputCars = await Console.readLineAsync(
+    "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+  );
+  const cars = inputCars.split(",").map((car) => car.trim());
+
+  inputCarsException(cars);
+  return cars;
+};
+
+const inputCount = async () => {
+  const count = await Console.readLineAsync("시도할 횟수는 몇 회인가요?\n");
+
+  inputCountException(count);
+  Console.print("\n");
+  return count;
+};
+
+const randomNum = () => {
+  return MissionUtils.Random.pickNumberInRange(0, 9);
+};
+
+const racingCar = async (cars, result, randomNumArr) => {
+  for (let i = 0; i < cars.length; i++) {
+    if (randomNumArr[i] >= 4) result[i] += 1;
+    Console.print(`${cars[i]} : ${"-".repeat(result[i])}`);
+  }
+  Console.print("\n");
+};
+
+const racingCount = async (count, cars) => {
+  let result = new Array(cars.length).fill(0);
+
+  for (let i = 0; i < count; i++) {
+    const randomNumArr = cars.map(() => randomNum());
+    racingCar(cars, result, randomNumArr);
```

Comment:
```suggestion
    const randomNumArr = cars.map(randomNum);
```
이건 좀 잡기술이긴 한데 `map` 함수에 단순히 값을 받아서 다른 값을 반환하는 함수를 전달할 때는 함수의 참조만 전달하는 것도 가능합니다!

개인적으로 배열 요소들을 숫자로 변환할 때 `map(Number)`와 같이 자주 사용하는 편입니다

Reply:
아 저렇게도 쓸 수 있군요
훨씬 간단하긴하네요ㅋㅋㅋㅋㅋㅋ 꿀Tip 감사합니다감사합니다👍🏻👍🏻